test/sequential/test-performance-eventloopdelay.js

```diff
@@ -97,3 +98,5 @@ const {
   }
   spinAWhile();
 }
+
+process.on('exit', global.gc);
```

Comment:
It's not obvious to me why this line is necessary in this test. Does it warrant a comment?

Reply:
Yeah, I’ve added a comment. It’s not necessary, but it serves as a test to make sure that tearing down the native class works during GC rather than only during process exit.